src/index.html

```diff
@@ -17,6 +29,87 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo_link"
+      >
+        <img
+          class="logo"
+          src="./images/logo.png"
+          alt="Moyo logo"
+        />
+      </a>
```

Comment:
The <a> tag's 'href' property should not contain spaces. If the 'href' is meant to be a placeholder, consider using a more descriptive placeholder, such as '#home' or '#logo'.